server/subsonic/jukebox.go

```diff
@@ -0,0 +1,193 @@
+package subsonic
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/navidrome/navidrome/core/playback"
+	"github.com/navidrome/navidrome/log"
+	"github.com/navidrome/navidrome/model"
+	"github.com/navidrome/navidrome/server/subsonic/responses"
+)
+
+type ActionType int
+
+const (
+	ActionUnknown ActionType = iota
```

Comment:
IMO, we don't need these constants as `iota`. We could simply set them to the string they represent, and in the big `switch` below, use `default` instead of `ActionUnknown`. This way we don't even need the `parseAction` function. Or am I missing something?

Reply:
You're totally right. This was way to complicated. Fixed:

[jukebox-mode 8d7739dd] Decomplicate action-parsing